package.json

```diff
@@ -5,12 +5,13 @@
   "dependencies": {
     "@highlightjs/cdn-assets": "^10.7.3",
     "blueimp-file-upload": "^10.31.0",
-    "bootstrap": "^3.4.1",
+    "bootstrap-sass": "^3.4.1",
     "font-awesome": "^4.7.0",
     "gulp": "^4.0.2",
     "gulp-if": "^3.0.0",
     "gulp-less": "^4.0.1",
     "gulp-rename": "^2.0.0",
+    "gulp-sass": "^4.1.0",
```

Comment:
gulp-sass 5.0.0 was released a 20 days ago, would be nice to update to that directly. Requires setting the compiler yourself https://www.npmjs.com/package/gulp-sass#user-content-migrating-to-version-5